src/components/AllEmployees/container.js

```diff
@@ -1,48 +1,69 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
 import { PropTypes as PT } from 'prop-types';
-import Swal from 'sweetalert2';
-import { getAllUsers } from '../../services/userService';
+import { getEmployeesLoading, getAllEmployees, getSelectedEmployeeId } from '../../store/reducers';
+import { updateSelectedEmployee } from '../../store/actions/employees';
 
 const AllEmployeesContainer = Wrapped =>
   class extends React.Component {
     static propTypes = {
+      updateSelectedEmployee: PT.func,
+      selectedEmployeeId: PT.oneOfType([
+        PT.string,
+        PT.number,
+      ]),
```

Comment:
String or number? Something doesn't seem right here

Reply:
The data goes in as a number. Formik converts it to a string onchange